packages/asset/contracts/Asset.sol

```diff
@@ -175,10 +179,7 @@ contract Asset is
     }
 
     function _setMetadataHash(uint256 tokenId, string memory metadataHash) internal {
-        require(
-            hasRole(MINTER_ROLE, _msgSender()) || hasRole(DEFAULT_ADMIN_ROLE, _msgSender()),
-            "Asset: must have minter or admin role to mint"
-        );
+        require(hasRole(MINTER_ROLE, _msgSender()), "Asset: must have minter or admin role to mint");
```

Comment:
Only MINTER_ROLE

Reply:
I will actually remove the role check in this place as it is check in all functions that use this one.